### DISCUSSION TITLE
OCR Transparency

### DISCUSSION THREAD

**Discussion by FiYir** [unrealities/MTGA_Draft_17Lands#20] [General]
@unrealities Your explanation of the new OCR feature could use some clarity. While I understand the need for discretion due to security concerns, the current description might leave users feeling uneasy about a program that captures their screen and sends it to a third-party service.

To address these concerns:
- Update the README with a dedicated section providing a comprehensive explanation of the feature. Specify that the app captures a screenshot only when clicking the refresh button if P1P1 (Pack 1 / Pick 1) hasn't been seen, stopping after P1P1. Clarify the use of the Google Cloud Vision API for identifying card names. The more details the better.
- Consider saving screenshots locally for user inspection. The app can automatically create a 'Screenshots' folder, holding all captured images for easy access.
- Think about adding a settings toggle for enabling/disabling the OCR functionality, similar to the existing draft logs setting. 

What do you think about these suggestions?

**Comment by unrealities** [comment]
I agree with everything. I'll work on updating the README immediately to help ease any concerns.

**Reply by unrealities** [reply]
README has been updated.
